test/archethic/mining/fee_test.exs

```diff
@@ -20,7 +20,7 @@ defmodule Archethic.Mining.FeeTest do
 
     test "should return a fee less than amount to send for a single transfer" do
       # 0.050048 UCO for 1 UCO at $0.2
-      assert 50_048_000 =
```

Comment:
Can you also change the comments values

Reply:
I have updated all fee comments with new value by dividing with `100000000`